src/permutat.cc

```diff
@@ -349,9 +353,18 @@ static Obj ProdPerm(Obj opL, Obj opR)
     const TR *          ptR;            /* pointer to the right operand    */
     UInt                p;              /* loop variable                   */
 
-    /* compute the size of the result and allocate a bag                   */
     degL = DEG_PERM<TL>(opL);
     degR = DEG_PERM<TR>(opR);
+
+    // Deal with trivial cases
```

Comment:
```suggestion
    // handle trivial cases
```
I prefer uniformity between the various functions whenever possible, that IMHO highlights the real differences.